Corona-Warn-App/src/main/AndroidManifest.xml

```diff
@@ -76,14 +77,6 @@
             android:exported="false"
             android:screenOrientation="fullSensor"
             tools:replace="screenOrientation" />
-        <activity
```

Comment:
So this PR needs to be merged after the TABBar PR, right?

Reply:
Yes